app.rb

```diff
@@ -0,0 +1,172 @@
+require './book'
+require './teacher'
+require './student'
+require './rental'
+
+class App
+  attr_reader :books, :people, :rental
+
+  def initialize
+    @books = []
+    @people = []
+    @rental = []
+  end
+
+  def run
+    show_menu
+
+    menu_list = {
+      '1' => method(:book_list),
+      '2' => method(:people_list),
+      '3' => method(:add_teacher),
```

Comment:
- [ ] It will be great to ensure when a user selects `3` he/she has the option of creating a new `person`. At the moment your implementation only allows for creating a teacher. Remember this option should read `Create a person and it should give you the option of creating either a `student` or `teacher`

Reply:
Option 3 is teacher
option 4 is student